### PR TITLE
Fix #982: Removes the view bug on first load

### LINKED ISSUE #982: Unnecessary Information on First Load
### Steps to reproduce
- Go to https://evalai.cloudcv.org/
- Refresh the page and see the top right corner.
OS : `Ubuntu 14.04`
Browser: `Chrome Version 57.0.2987.133 (64-bit)`
### ` Expected `

correct

Thanks!

### PR DESCRIPTION
**Pull request by aayusharora**
- No element in DOM for `Logout` button till the user is authenticated.
- Replaced `ng-show/hide` by `ng-if`.

@deshraj @aka-jain Kindly review. :)
------ Signed off by @angularboy--------

### CODE REVIEW

File: frontend/src/views/web/partials/main-header.html [hunk 1]
```diff
@@ -28,12 +28,13 @@
...
-                <li ng-hide="isAuth"><a class="waves-effect waves-dark main-header-link" ui-sref="auth.login" ui-sref-active="active"><i class="fa fa-sign-in"></i>Log In</a></li>
+                <li ng-if="!isAuth"><a class="waves-effect waves-dark main-header-link" ui-sref="auth.signup" ui-sref-active="active"><i class="fa fa-sign-out"></i>Sign Up</a></li>
+                <li ng-if="!isAuth"><a class="waves-effect waves-dark main-header-link" ui-sref="auth.login" ui-sref-active="active"><i class="fa fa-sign-in"></i>Log In</a></li>
                 <!-- show after authenticate user -->
                 <li ng-show="isAuth"><a class="waves-effect waves-dark main-header-link" ui-sref="web.dashboard" ui-sref-active="active"><i class="fa fa-thumb-tack"></i>Dashboard</a></li>
                 <li ng-show="isAuth"><a class="waves-effect waves-dark main-header-link" ui-sref="auth.signup" ui-sref-active="active"><i class="fa fa-user"></i>Hi {{header.user.name}}!</a></li>
             </ul>
         </div>
     </nav>
+    <!--...............................................................-->
```
Comment (deshraj) [inline frontend/src/views/web/partials/main-header.html]: Can you please remove this comment?

File: frontend/src/views/web/partials/main-header.html [hunk 2]
```diff
@@ -5,18 +5,18 @@
...
+                <li ng-if="!isAuth"><a class="waves-effect waves-dark main-header-link" ui-sref="auth.signup" ui-sref-active="active">Sign Up</a></li>
+                <li ng-if="!isAuth"><a class="btn ev-btn-dark waves-effect waves-dark grad-btn grad-btn-dark fs-14" ui-sref="auth.login" ui-sref-active="active">Log In</a></li>
                 <!-- show after authenticate user -->
-                <li ng-show="isAuth">
-                    <a ng-click="header.profileDropdown($event)" class="dropdown-button waves-effect waves-dark main-header-link" data-activates='ev-dropdown' ui-sref-active="active">Hi <strong>{{header.user.name}}</strong> &nbsp; 
-          <!-- <i class="fa fa-caret-down text-light-gray icon"></i> -->
-          </a>
+                <li ng-if="isAuth">
+                    <a ng-click="header.profileDropdown($event)" class="dropdown-button waves-effect waves-dark main-header-link" data-activates='ev-dropdown' ui-sref-active="active">Hi <strong>{{header.user.name}}</strong> &nbsp;
+                        <!-- <i class="fa fa-caret-down text-light-gray icon"></i> -->
```
Comment (deshraj) [inline frontend/src/views/web/partials/main-header.html]: Can you please remove this comment?

**Comment by aayusharora** [comment]
Done @deshraj Thanks for the review.

**Comment by aka-jain** [comment]
LGTM, Thannks @aayusharora 
@deshraj let me know when you done with your review.

**Comment by deshraj** [comment]
This looks good to me. Merging it. Thanks @aayusharora :)

**Pull request merged**